11-african-bush-elephant/technical_specification.md

```diff
@@ -0,0 +1,58 @@
+# Metadata Submission via Data Portal UI (African Bush Elephant)
+
+## The Scope
+
+### Summary:
+
+The aim of this epic is to add new features and improve existing features of the Data portal UI's interaction with the Metadata Repository Service.
+
+### Included/Required:
+
+- Visualisation of the metadata schema in the Data portal
+- Download, validate and submission of spreadsheet via Data portal
+- Download of the transformed spreadsheet in JSON format via Data Portal
+- Submission process of metadata via Data portal
+- Extend Metadata Repository Service API to include endpoints used by data portal for
+  Dataset summary and Metadata summary.
+
+### Not included:
+
+- user authentication & authorization
+- restricted/encrypted metadata
+
+## User Journeys
+
+This epic covers the following user journeys:
+
+![](user_journey_metadata_schema_visualisation.jpg)
```

Comment:
Looks good. Just one small comment: Could you describe each step so that it is clear what is expected and what needs to be implemented? 

Take a look at other Epics for examples on how to elaborate each step in the user journey.